calm/samples/traderx/flows/trade-processing/trade-processing-new-trade.json

```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "https://calm.finos.org/draft/2024-10/meta/flow.json",
+  "$id": "https://calm.finos.org/traderx/flows/trade-processing.json",
+  "unique-id": "flow-trade-processing",
+  "name": "Trade Processing - New Trade",
+  "description": "The process flow for handling new trade events",
+  "transitions": [
+    {
+      "relationship-unique-id": "trade-processor-connects-to-trade-feed",
+      "sequence-number": 1,
+      "summary": "New Trade Request (trades/new)",
+      "direction": "destination-to-source"
+    },
+    {
+      "relationship-unique-id": "trade-processor-connects-to-traderx-db",
+      "sequence-number": 2,
+      "summary": "Insert New Trade"
+    },
+    {
+      "relationship-unique-id": "trade-processor-connects-to-trade-feed",
+      "sequence-number": 3,
+      "summary": "Publish New TradeEvent (accounts/$id/trades)"
+    },
+    {
+      "relationship-unique-id": "trade-processor-connects-to-trade-feed",
+      "sequence-number": 3,
```

Comment:
Is the use of the same sequence number inticating a parallel flow? If so this answers my other comment, although wonder if it would be nicer to be able to specify arrays of transitions which would imply the same?

Reply:
Yes that was my intention here

Although on UML sequence https://github.com/finos/traderX/blob/main/docs/flows.md#4-trade-processing it represents as two separate flows back. The reality is both are happening at same time.

Now in traderx the relationships of trades and positions between trade feed and web gui are maintained separately as assume even though logically the same connection between components, the API interface are completely different.  Perhaps this should be one logical relationship and therefore this whole things get simplified and we drop the sequence number